common/events.h

```diff
@@ -97,10 +97,12 @@ typedef uint32 CustomEventType;
 struct Event {
 	/** The type of the event. */
 	EventType type;
-	/** Flag to indicate if the event is real or synthetic. E.g. keyboard
-	  * repeat events are synthetic.
-	  */
-	bool synthetic;
+	/**
```

Comment:
Since this header is being modified anyway, would you mind fixing these doxygen comments in a separate commit to have appropriate whitespace between the members and at the start of each comment line? My inner perfectionist thanks you in advance.

Reply:
My OCD now feels better as well